### PR TITLE
Update docstring for base KPL

### PR DESCRIPTION
**Pull request by ianstenbit**
This is an example doc toward #1343

### CODE REVIEW

**Review by LukeWood — APPROVED** [review]
Thanks for the update Ian!

File: keras_cv/layers/preprocessing/base_image_augmentation_layer.py [hunk 1]
```diff
@@ -57,14 +57,23 @@ class BaseImageAugmentationLayer(tf.keras.__internal__.layers.BaseRandomLayer):
...
-    2. A dict of tensors with stable keys. The supported keys are:
-      `"images"`, `"labels"` and `"bounding_boxes"` at the moment. We might add
-      more keys in future when we support more types of augmentation.
-
-    The output of the `call()` will be in two formats, which will be the same
-    structure as the inputs.
+    The `call()` method supports two formats of inputs:
+    1. A single image tensor with shape (height, width, channels) or
+       (batch_size, height, width, channels)
+    1. A dict of tensors with stable keys. The supported keys are:
```
Comment (jbischof) [inline keras_cv/layers/preprocessing/base_image_augmentation_layer.py:63]: What does "stable" mean in this context?

**Review by jbischof — APPROVED** [review]
Nice!

File: keras_cv/layers/preprocessing/base_image_augmentation_layer.py (hunk 1 above)
Comment (ianstenbit) [reply keras_cv/layers/preprocessing/base_image_augmentation_layer.py]: I don't think this was the right word choice -- I've updated the wording a bit.

**Pull request merged**